etc/weblate/settings.py

```diff
@@ -256,6 +256,15 @@
 SOCIAL_AUTH_GITLAB_SECRET = os.environ.get('WEBLATE_SOCIAL_AUTH_GITLAB_SECRET', '')
 SOCIAL_AUTH_GITLAB_SCOPE = ['read_user']
 
+if 'WEBLATE_SOCIAL_AUTH_AUTH0_KEY' in os.environ:
+    AUTHENTICATION_BACKENDS += ('social_core.backends.auth0.Auth0OAuth2',)
+    SOCIAL_AUTH_AUTH0_SCOPE = ['openid', 'profile', 'email']
```

Comment:
You're not using `WEBLATE_SOCIAL_AUTH_AUTH0_KEY` here, is that intentional?

Reply:
ouch, you are right. I've coppied this from openshift config and missed this. Will fix it

Reply:
Maybe this would be actually nicer to be as it was in `settings_openshift.py` in weblate project? There was a function `import_env_vars` in `openshiftlib.py` which did this replacement from `WEBLATE_SOME_VAR` to `SOME_VAR`. Wouldn't that makes sense also here? So you woud not need to map all of thoes var by hand ...

Reply:
I'm not sure it's good idea, the problem is with some inconsistencies we have in the settings - some django variables are prefixed with `WEBLATE_` while some not.

In long term, most of the settings should be moved to the database and be configurable in the UI, but this is not a priority for now.